cmd/kapacitor/main.go

```diff
@@ -542,72 +541,15 @@ var (
 	dtype       = defineFlags.String("type", "", "The task type (stream|batch)")
 	dtemplate   = defineFlags.String("template", "", "Optional template ID")
 	dvars       = defineFlags.String("vars", "", "Optional path to a JSON vars file")
+	dfile       = defineFlags.String("file", "", "Optional path to a YAML or JSON template task file")
 	dnoReload   = defineFlags.Bool("no-reload", false, "Do not reload the task even if it is enabled")
-	ddbrp       = make(dbrps, 0)
+	ddbrp       = make(client.DBRPs, 0)
 )
 
 func init() {
 	defineFlags.Var(&ddbrp, "dbrp", `A database and retention policy pair of the form "db"."rp" the quotes are optional. The flag can be specified multiple times.`)
 }
 
-type dbrps []client.DBRP
-
-func (d *dbrps) String() string {
-	return fmt.Sprint(*d)
-}
-
-// Parse string of the form "db"."rp" where the quotes are optional but can include escaped quotes
-// within the strings.
-func (d *dbrps) Set(value string) error {
```

Comment:
Why was this moved to the client package? The `Set` method is part of the `flag.Value` interface, which seems like it belongs here in the main package.

Reply:
`client.TaskVars` ended up needing the same DBRP logic so rather than having two copies of the type, one in the `main` package and one in the `client` package I figured that I'd move it into client. I'm happy to have two separate types if you think thats best.

Reply:
Why did the `client.TaskVars` need to implement the `flag.Value` interface?

Reply:
Maybe the type in the client package should implement a TextUnmarshaler interface and then the type in the main package uses the type from the client package and adds the Set method to its own type? Does that work?

Reply:
Yeah I like that.

Reply:
@nathanielc Looks like implementing `TextUnmarshaler` causes issues with the regular JSON marshaling. Does having a method `Unmarshal` on the `client.DBRP` type work?